[FLINK-21723] Remove link to Hadoop integration from downloads page

tillrohrmann opened the pull request:
Yarn is no longer the only resource provider Flink users can use. Hence,
we should no longer directly point from the downloads page to this one
resource provider implementation.

zentol reviewed (approved):
+1, but it would be good to figure out where all the whitespace changes in the .md files came from. (If you modified them in IntelliJ, is the Save Actions plugin possible modifying them?)

tillrohrmann commented:
Thanks for the review @zentol. It must have been my Atom editor which added these whitespaces. I will revert the whitespace changes and then I'll merge this PR.

tillrohrmann commented:
Hmm, when looking at the whitespace changes, I noticed that we have a couple of lines which end with a whitespace and then with a `\n` character. I think this is what Atom has corrected. Hence I would think that this change is actually correct. I will separate it into a a hotfix commit though.

tillrohrmann committed:
[hotfix] Remove unnecessary whitespace at the end of lines

tillrohrmann committed:
[FLINK-21723] Remove link to Hadoop integration from downloads page

Yarn is no longer the only resource provider Flink users can use. Hence,
we should no longer directly point from the downloads page to this one
resource provider implementation.

This closes #425.

tillrohrmann committed:
Rebuild website
